config/config.go

```diff
@@ -10,16 +10,16 @@ import (
 	v2 "github.com/chanzuckerberg/fogg/config/v2"
 	"github.com/chanzuckerberg/fogg/errs"
 	"github.com/chanzuckerberg/fogg/util"
+	goVersion "github.com/hashicorp/go-version"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/afero"
 	"gopkg.in/yaml.v3"
-	goVersion "github.com/hashicorp/go-version"
 )
 
 var defaultTerraformVersion = goVersion.Must(goVersion.NewVersion("0.12.5"))
 
```

Comment:
I would probably move the `defaultFoggVersion` const over here, along with the `defaultTerraformVersion`

Reply:
Gotcha, thank you!